lib/compat/wordpress-6.5/fonts/class-wp-font-collection.php

```diff
@@ -203,6 +251,7 @@ private function load_from_url( $url ) {
 		 * @return array|WP_Error Array of data if valid, otherwise a WP_Error instance.
 		 */
 		private function validate_data( $data ) {
```

Comment:
Rename to `validate_and_sanitize` data for clarity?

Reply:
renamed here: https://github.com/WordPress/gutenberg/pull/58636/commits/d07b787b1a931cf5a8e32789122069f38447850b